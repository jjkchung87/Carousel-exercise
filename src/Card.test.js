import React from 'react';
import { render } from '@testing-library/react';
import Card from './Card';
import TEST_IMAGES from "./_testCommon.js";

//caption, src, currNum, totalNum

const idx = 1
const testImage = TEST_IMAGES[idx]


it('should render', () => {
    render(<Card caption={testImage.caption} src={testImage.src} currNum={idx} totalNum={TEST_IMAGES.length} />)
})

it('should match snapshot', () => {
    const {asFragment} = render(<Card caption={testImage.caption} src={testImage.src} currNum={1} totalNum={TEST_IMAGES.length} />)
    expect(asFragment()).toMatchSnapshot();
})