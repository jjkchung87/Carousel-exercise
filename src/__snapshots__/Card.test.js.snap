// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      testing image 2
    </h4>
    <img
      alt="testing image 2"
      class="Card-image"
      src="test2.com"
    />
    <small
      class="Card-small"
    >
      Image 1 of 3.
    </small>
  </div>
</DocumentFragment>
`;
